docs/docs/articles/test-workflows-examples-expressions.md

```diff
@@ -0,0 +1,53 @@
+# Test Workflows Examples - Expressions
+
+## Expressions Language
+
+We have designed a simple expressions language, that allows dynamic evaluation of different values.
+
+## JSON-Native
+
+It is built on JSON, so every JSON syntax is a valid expression value as well, like **[ “a”, “b”, “c” ]**.
```

Comment:
Quotes are invalid, as these are not regular double quotes, so after copying it won't work. Probably better to just wrap in code:
```suggestion
It is built on JSON, so every JSON syntax is a valid expression value as well, like `[ "a", "b", "c" ]`.
```